dashboard/requirements.in

```diff
@@ -1,6 +1,28 @@
-jsonschema
-requests
-matplotlib
+pycodestyle
 boto3
-mako
+botocore
+certifi
+chardet
 fastlog
+cycler
+docutils
+idna
+jmespath
+jsonschema
+mako
+markupsafe
+matplotlib
+numpy
+pyparsing
+python-dateutil
+pytz
+requests
+s3transfer
+six
+urllib3
+svgwrite
+radon
+ansi2html
+pyrebase
+pydocstyle
+vulture
```

Comment:
Please add only requirements that relevant to the development and deployment phase in this file. I dont think even vulture is required for development

Reply:
We need vulture for coverage usually so all the repos have the qa folder so we go to that folder and run the script for coverage , refer src/external-tools.py